linux-tkg-config/prepare

```diff
@@ -506,23 +506,24 @@ _tkg_srcprep() {
     cd ${wrksrc}/linux-${_kern_ver}
   fi
 
-  
-  if [ -z "${_configfile}" ]; then    
+
+  if [ -z "${_configfile}" ]; then
     if [ "${_distro}" = "Arch" ] || [ "$_distro" = "Void" ]; then
       cat "${srcdir}"/config.x86_64 > ./.config
     else
       if [ -f /boot/config-`uname -r` ];then
-        _msg="Using /boot/config-`uname -r` as config file"
+        msg2 "Using /boot/config-`uname -r` as config file"
         cp /boot/config-`uname -r` .config
       elif [ -f /proc/config.gz ];then
-        _msg="Using /proc/config.gz as config file"
+        msg2 "Using /proc/config.gz as config file"
         zcat --verbose /proc/config.gz > .config
       else
-        _msg="Current kernel config not found! Falling back to default..."
+        msg2 "Current kernel config not found! Falling back to default..."
       fi
     fi
   else
-    cat "${srcdir}/${_configfile}" > ./.config
+    msg2 "Using user-provided config file in ${_where}/${_configfile}"
```

Comment:
Okay, here if one uses `config_hardened.x86_64` it will still say this sentence. Maybe I should make a condition ?